### PR TITLE
Url Enrichment - Generic

### PR DESCRIPTION
**Pull request by yaakovi**
updated playbook for version 3.5.0

### CODE REVIEW

File: Playbooks/playbook-URL_Enrichment_-_Generic.yml [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+      }
+    }
+  }
+inputs:
+- key: url
+  value: {}
+  required: true
+  description: url to enrich
+outputs:
+- contextPath: URL.Address
```
Comment (JonathanMeler) [inline Playbooks/playbook-URL_Enrichment_-_Generic.yml]: Add the `URL` output as well (with out it, if any integration add other attributes like `Malicious` it won't return to the main playbook)

File: TestPlaybooks/playbook-Url_Enrichment_-_Generic_Test.yml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      version: -1
+      name: URL Enrichment - Generic
+      playbookName: URL Enrichment - Generic
+      type: playbook
+      iscommand: false
+      brand: ""
+    scriptarguments:
+      url:
+        simple: ${URL.Data}
+    separatecontext: false
```
Comment (JonathanMeler) [inline TestPlaybooks/playbook-Url_Enrichment_-_Generic_Test.yml]: Missing output  "Assert" to check results. use `VerifyContext` command

File: Playbooks/playbook-URL_Enrichment_-_Generic.yml (hunk 1 above)
Comment (yaakovi) [reply Playbooks/playbook-URL_Enrichment_-_Generic.yml]: added malicious and its sub-labels (vendor, description)

Comment (JonathanMeler) [reply Playbooks/playbook-URL_Enrichment_-_Generic.yml]: please add `URL` as well. o/w the data won't return from relevant integrations

**Comment by JonathanMeler** [comment]
fixes: https://github.com/demisto/etc/issues/10315

**Pull request merged**